### ISSUE TITLE
[Request] QCollapsible support for routes

### ISSUE THREAD

**Issue by AlexRadch** [quasarframework/quasar#937]
### Software version

Quasar: last
OS: Windows
Node: last
NPM: last
Browsers: last Firefox
iOS: not tested
Android: not tested
Any other software related to your bug: none

### What did you get as the error?

I created collapsible menu like next:
```html
<q-list>
  <q-collapsible group="/" label="Menu1" to="/Menu1/">
    <q-item to="/Menu1/Item11/">Item11</q-item>
    <q-item to="/Menu1/Item12/">Item12</q-item>
  </q-collapsible>
  <q-collapsible group="/" label="Menu2" to="/Menu2/">
    <q-item to="/Menu2/Item21/">Item21</q-item>
    <q-item to="/Menu2/Item22/">Item22</q-item>
  </q-collapsible>
</q-list>
```

Where QCollapsible have `to` property. I want to select not only `QItem` items but `QCollapsible` items also to select "/Menu1/" and "/Menu2/" pages. 

### What were you expecting?

I want supporting QCollapsible selection like http://www.muse-ui.org/#/list does. See `nested list` and `selectable list` sections on english page.

### What steps did you take, to get the error?

I created collapsible menu where all items and sub items should support selection.

**Comment by rstoenescu** [comment]
You can see nested lists in the showcase. Look at its source code if you want.

Selectable you can program yourself. Due to the possible variance and complexity in use cases, it's best each dev do its own.

Will add support for vue router on QCollapsible for v0.15.

**Comment by FerrielMelarpis** [comment]
Hi @rstoenescu, do you have a workaround for dynamically opening a `q-collapsible` when its `q-side-link` child has `linkExactActiveClass` from Vue Router? I'm currently using quasar v0.14.

**Comment by FerrielMelarpis** [comment]
nvm, got a workaround using events

**Comment by slowaways** [comment]
Raz, Milestone 0.17.x?
It's priority.

**Comment by rstoenescu** [comment]
Available in future Quasar v0.17.9

**Comment by slowaways** [comment]
Omg! Yeah!!
